webapp/src/modules/wallet/utils.ts

```diff
@@ -22,7 +22,7 @@ export function shortenAddress(address: string) {
 }
 
 export function addressEquals(address1?: string, address2?: string) {
-  return address1 !== undefined && address2 !== undefined && address1.toLowerCase() === address2.toLowerCase()
+  return !!address1 && !!address2 && address1.toLowerCase() === address2.toLowerCase()
```

Comment:
What do you think of doing:
```suggestion
  return address1?.toLowerCase() === address2?.toLowerCase()
```
It would allow us to compare two null or undefined strings as well